<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQInfoUI_KDZ</class>
 <widget class="QWidget" name="QQInfoUI_KDZ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1430</width>
    <height>990</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQ资料</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1430</width>
     <height>990</height>
    </rect>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QLabel" name="label_myID">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>150</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>账号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_myname">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>250</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>昵称:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mysignature">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>350</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>个性签名:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mysex">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>450</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>性别:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_myarea">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>550</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>地区:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mybirthday">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>650</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>生日:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_myQQyear">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>740</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Q龄:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_myname">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>250</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mysignature">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>350</y>
       <width>500</width>
       <height>60</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mysex">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>450</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_myarea">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>550</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_mybirthday">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>650</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Arial&quot;;</string>
     </property>
     <property name="date">
      <date>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="label_myQQyearOUT">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>745</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial Narrow&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_myIDOUT">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>150</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>990</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/back1.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_save">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>860</y>
       <width>600</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>保 存 资 料</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_myinfo">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>30</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>个人资料</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGraphicsView" name="graphicsView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>990</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/back2.png);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_friendinfo">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>30</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>好友资料</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_FID">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>150</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>账号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Fname">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>250</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>昵称:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Fsignature">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>350</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>个性签名:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Fsex">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>450</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>性别:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Farea">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>550</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>地区:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Fbirthday">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>650</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>生日:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FQQyear">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>750</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Q龄:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FIDOUT">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>150</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FnameOUT">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>250</y>
       <width>300</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FsignatureOUT">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>350</y>
       <width>350</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FsexOUT">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>450</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FareaOUT">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>550</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FbirthdayOUT">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>655</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FQQyearOUT">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>755</y>
       <width>150</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Fsend">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>860</y>
       <width>300</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>发消息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Fdelete">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>860</y>
       <width>300</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>940</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/back3.png);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GID">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>210</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>群号码:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Gname">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>520</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>群名称:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Gcreater">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>360</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>群主:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_groupinfo">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>70</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>群资料</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_GIDOUT">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>210</y>
       <width>250</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GcreaterOUT">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>360</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Gdelete">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>800</y>
       <width>300</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>退出/解散</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Gsend">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>800</y>
       <width>300</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>发消息</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_GnameOUT">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>520</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Gchange">
     <property name="geometry">
      <rect>
       <x>1230</x>
       <y>525</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
